### PR TITLE
Bugfix: Do not allow empty Tags in create-tag Dialog 

### PR DESCRIPTION
**Pull request by pKallert**
**What I did**
When creating new Tags, the UI allows for empty tags to be created and then throws an error. 

**How I did it**
Just like in the UploadDialog I disabled the create button until there is an actual value entered in the CreateTag Dialog. Also, the enter button on the keyboard does not work anymore until the field is not empty anymore.

**How to verify it**
When creating a new Tag, the 'create' button is disabled and 'enter' does not create  new tag until an actual value is entered into the textfield.

### CODE REVIEW

**Comment by pKallert** [comment]
closes #108

File: Resources/Private/JavaScript/media-module/src/components/Dialogs/CreateTagDialog.tsx [hunk 1]
```diff
@@ -66,7 +66,7 @@ const CreateTagDialog: React.FC = () => {
                     type="text"
                     value={dialogState.label}
                     onChange={setLabel}
-                    onEnterKey={handleCreate}
+                    onEnterKey={createPossible ? handleCreate : null}
```
Comment (Sebobo) [inline Resources/Private/JavaScript/media-module/src/components/Dialogs/CreateTagDialog.tsx:69]: Thx for the improvemt! But did you forget to set the disabled state like you mentioned in the description of the PR?

**Review by Sebobo — CHANGES_REQUESTED** [review]

File: Resources/Private/JavaScript/media-module/src/components/Dialogs/CreateTagDialog.tsx (hunk 1 above)
Comment (pKallert) [reply Resources/Private/JavaScript/media-module/src/components/Dialogs/CreateTagDialog.tsx:69]: With disable state I meant disabling the button at the beginning. This is done by setting createPossible to something other than true.

Comment (Sebobo) [reply Resources/Private/JavaScript/media-module/src/components/Dialogs/CreateTagDialog.tsx:69]: Ah sorry, you're right. The variable was already there and in use but just not dynamic.

**Review by Sebobo — APPROVED** [review]

**Pull request merged**